nexus/db-queries/src/authz/api_resources.rs

```diff
@@ -869,7 +869,8 @@ authz_resource! {
 authz_resource! {
     name = "LoopbackAddress",
     parent = "Fleet",
-    primary_key = Uuid,
+    // TODO: make it easier to specify the typed UUID kind
```

Comment:
Does this still need work? I'm not 100% sure I follow what additional work is needed here

Reply:
Nothing else needed here.